Corrfunc/io.py

```diff
@@ -19,7 +19,7 @@
 __all__ = ('read_fastfood_catalog', 'read_ascii_catalog', 'read_catalog')
 
 
-def read_fastfood_catalog(filename, return_dtype=None, need_header=None):
+def read_fastfood_catalog(filename, return_dtype=None, need_weights=None, need_header=None):
```

Comment:
I don't see `need_header` being used anywhere in the code-base - should we remove this keyword?